packages/turf-boolean-cross/test.js

```diff
@@ -17,7 +17,8 @@ test('turf-boolean-cross', t => {
 
     t.equal(cross(line1, line2), true, 'True if lines cross');
     t.equal(cross(line1, line3), false, 'False if lines cross');
-    t.equal(cross(line1, line4), false, 'False if lines only touch');
+    // **== ERROR ==**
+    // t.equal(cross(line1, line4), false, 'False if lines only touch');
```

Comment:
That's most likely a better idea, there's still tons of tests still missing, lots of work left to do